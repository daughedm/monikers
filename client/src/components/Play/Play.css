.background-image {
  background-image: url('../../assets/seamlessbg-03.svg');
  background-repeat: repeat;
  background-size: 75%;
}

.play {
  width: 100%;
  height: 100vh;
  
}

.game-info-container {
  background-color: #FFF;
  width: 100%;
  display: flex;
  align-content: space-between;
  justify-content: space-between;
  margin: auto;
  padding: 0 20px;
}

.current-team {
  color: #00B4EF;
  margin: 10px 0 10px 0;
}

.current-round {
  color: #00B4EF;
  margin: 10px 0 10px 0;

}

.card-title {
  text-align: center;
  color: #555;
  margin: 0 auto 20px auto;
  font-size: 1.5rem;
  line-height: 1.6rem;
}

.points {
  text-align: center;
  color: #FFF;
  font-size: 1.4rem;
}

.description {
  color: #555;
  font-size: 0.9rem;
  line-height: 1.2rem;

}

.category {
  text-align: center;
  font-size: 1rem;
  margin: 10px auto;

}

.card-container {
  position: relative;
  background-color: #FFF;
  width: 85%;
  height: 410px;
  margin: 20px auto;
  padding: 30px 25px 0 25px;
  border-radius: 15px;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
}

.ripple-pass {
  background-position: center;
  transition: background 0.8s;
}

.ripple-pass:hover {
  background: rgba(239, 83, 63, 1) radial-gradient(circle, transparent 1%, 
  rgba(239, 83, 63, 1) 1%) center/15000%;
}

.ripple-pass:active {
  background-color: rgba(239, 83, 63, 0.75);
  background-size: 100%;
  transition: background 0s;
}

.ripple-got-it {
  background-position: center;
  transition: background 0.8s;
}

.ripple-got-it:hover {
  background: rgba(76, 189, 159, 1) radial-gradient(circle, transparent 1%, 
  rgba(76, 189, 159, 1) 1%) center/15000%;
}

.ripple-got-it:active {
  background-color: rgba(239, 83, 63, 0.75);
  background-size: 100%;
  transition: background 0s;
}

.pass-button {
  width: 48%;
  height: 40px;
  color: #FFF;
  background-color: rgba(239, 83, 63, 1);
  border: 1px solid #FFF;
  border-radius: 5px;
  font-size: 1rem;
  margin-top: 0;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
}

.got-it-button {
  width: 48%;
  height: 40px;
  color: #FFF;
  background-color: rgba(76, 189, 159, 1);
  border: 1px solid #FFF;
  border-radius: 5px;
  font-size: 1rem;
  margin-top: 0;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
}

.dashed-line {
  border-bottom: 2px dotted #999;
  width: 40%;
  margin: 20px auto;
}

.circle {
  text-align: center;
  width: 50px;
  height: 50px;
  margin: auto;
  display:flex;
  align-items: center;
  justify-content: center;
  height:45px;
  width:90px;
  border-radius: 90px 90px 0 0;
  position: absolute;
  bottom: 0;
  left: 0;
  right: 0;

}

.buttons-container {
  display: flex;
  align-content: space-between;
  justify-content: space-between;
  width: 85%;
  position: fixed;
  left: 0;
  right: 0;
  bottom: 25px;
  margin: auto;

}

.timer {
  height: 10px;
  width: 100%;
  margin: auto;
  background-color: rgba(76, 189, 159, 1);
  animation-name: countdown;
  animation-duration: 60s;
}

@keyframes countdown {
  0%   {
    background-color: rgba(76, 189, 159, 1);
    width: 100%;
  }
  50%  {
    background-color: rgb(240, 221, 49);
    width: 50%;
  }
  100% {
    background-color: rgba(239, 83, 63, 1);
    width: 0%;
  }
}

@media (min-width: 375px) {
  .card-container {
    height: 71%;
    margin: 30px auto;
    padding: 30px 30px 0 30px;
  }

  .description {
    color: #555;
    font-size: 1.1rem;
    line-height: 1.6rem;
  }

  .dashed-line {
    margin: 30px auto;
  }

  .buttons-container {
    bottom: 5%;
  }
  
  .category {
    font-size: 1.1rem;
  }

}

@media (min-width: 411px) {
  
.pass-button {
  height: 45px;
}

.got-it-button {
  height: 45px;
}

.current-team {
  margin: 15px 0 15px 0;
}

.current-round {
  margin: 15px 0 15px 0;

}

}

@media (min-width: 620px) {
.play {
  width: 400px;
  margin: auto;
}

.buttons-container {
  width: 340px;
}

.card-container {
  width: 340px;
  height: 500px;
  margin: 20px auto;
  padding: 30px 25px 0 25px;
}

.buttons-container {
  display: flex;
  align-content: space-between;
  justify-content: space-between;
  width: 85%;
  position: static;
  margin: auto;

}
}