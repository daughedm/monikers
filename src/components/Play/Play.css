.play {

}

.game-info-container {
  background-color: #FFF;
  width: 100%;
  display: flex;
  align-content: space-between;
  justify-content: space-between;
  margin: auto;
  padding: 0 20px;
}

.current-team {

}

.current-rouund {

}

.card-title {
  text-align: center;
  color: #555;
  margin: 10px auto 20px auto;
}

.points {
  text-align: center;
  color: #FFF;
  font-size: 1.4rem;
}

.description {
  color: #555;
  font-size: 1.1rem;

}

.category {
  text-align: center;
  color: #866AAD;

}

.card-container {
  background-color: #FFF;
  width: 80%;
  margin: 30px auto;
  padding: 20px 25px 40px 25px;
  border-radius: 15px;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
}

.ripple-pass {
  background-position: center;
  transition: background 0.8s;
}

.ripple-pass:hover {
  background: rgba(239, 83, 63, 1) radial-gradient(circle, transparent 1%, 
  rgba(239, 83, 63, 1) 1%) center/15000%;
}

.ripple-pass:active {
  background-color: rgba(239, 83, 63, 0.75);
  background-size: 100%;
  transition: background 0s;
}

.ripple-got-it {
  background-position: center;
  transition: background 0.8s;
}

.ripple-got-it:hover {
  background: rgba(76, 189, 159, 1) radial-gradient(circle, transparent 1%, 
  rgba(76, 189, 159, 1) 1%) center/15000%;
}

.ripple-got-it:active {
  background-color: rgba(239, 83, 63, 0.75);
  background-size: 100%;
  transition: background 0s;
}

.pass-button {
  width: 48%;
  height: 50px;
  color: #FFF;
  background-color: rgba(239, 83, 63, 1);
  border: 1px solid #FFF;
  border-radius: 5px;
  font-size: 1rem;
  margin-top: 0;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
}

.got-it-button {
  width: 48%;
  height: 50px;
  color: #FFF;
  background-color: rgba(76, 189, 159, 1);
  border: 1px solid #FFF;
  border-radius: 5px;
  font-size: 1rem;
  margin-top: 0;
  -webkit-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  -moz-box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
  box-shadow: 0px 2px 12px 0px rgba(0,0,0,0.37);
}

.dashed-line {
  border-bottom: 1px dashed #555;
  width: 40%;
  margin: 30px auto;
}

.circle {
  border-radius: 100%;
  background-color: #866AAD;
  text-align: center;
  width: 50px;
  height: 50px;
  margin: auto;
  display:flex;
  align-items: center;
  justify-content: center;
}

.buttons-container {
  display: flex;
  align-content: space-between;
  justify-content: space-between;
  width: 80%;
  margin: auto;

}

