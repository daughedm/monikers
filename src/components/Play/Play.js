import React, { Component } from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';
import * as actions from '../../actions';
import './Play.css';

export class Play extends Component {
  constructor() {
    super();
  }
  componentDidMount() {}

  render() {
    return (
      <div className="play">
        <div className="game-info-container">
          <h3 className="current-team">Blue Team</h3>
          <h3 className="current-round">Round 1</h3>

        </div>
        {this.props.cards.length && 
        <div className="card-container">
          <h1 className="card-title">{this.props.cards[0].name}</h1>

          <p className="description">{this.props.cards[0].description}</p>
          <div className="dashed-line" ></div>
          <h3 className="category">{this.props.cards[0].category}</h3>
          {/* this divs colors will change based on category */}
          <div className="circle">
            <h1 className="points">{this.props.cards[0].pointValue}</h1>
          </div>
        </div>
        }
        <div className="buttons-container">
          <button className="pass-button ripple-pass">Pass</button>
          <button className="got-it-button ripple-got-it">Got It!</button>
        </div>
      </div>
    );
  }
}

export const mapStateToProps = state => ({
  cards: state.cards
});

export const mapDispatchToProps = dispatch => ({});

Play.propTypes = {};

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(Play);
