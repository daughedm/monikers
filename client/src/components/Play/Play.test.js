/* eslint-disable max-len */

import React from 'react';
import ReactDOM from 'react-dom';
import Play from './Play';

it('renders without crashing', () => {
  expect(true).toEqual(true);
});
