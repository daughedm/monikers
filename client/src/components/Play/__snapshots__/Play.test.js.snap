// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`play matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Play
    activeCards={
        Array [
            Object {
              "category": "CELEBRITY",
              "description": "An Internet meme that shows a Shiba Inu surrounded by colorful Comic Sans text that describes its inner monologue, such as \\"Wow,\\" \\"Concern,\\" and \\"so scare.\\" There is much confuse over the name's pronunciation, yet it was recently used to brand a Bitcoin competitor.",
              "name": "Doge",
              "pointValue": 3,
            },
          ]
    }
    addCard={[Function]}
    currRound={1}
    currTeam="team1"
    currentRound={[Function]}
    currentTeam={[Function]}
    discardPile={
        Array [
            Object {
              "category": "ET CETERA",
              "description": "A virus that causes hemorrhagic fever, typically acquired by contact with an infected monkey, fruit bat, or person. Its most severe symptoms can include bleeding from the eyes, nose, ears, mouth, and rectum. There is no specific treatment for the disease.",
              "name": "Ebola",
              "pointValue": 2,
            },
            Object {
              "category": "CELEBRITY",
              "description": "A Japanese competitive eater who shocked the world in 2001 by eating 50 hot dogs and buns (HDB) in 12 minutes at Nathan's Hot Dog Eating Contest, doubling the previous record. He once lost a hot dog eating contest (no buns) to a 1089 lb. Kodiak bear (31-50).",
              "name": "Kobayashi",
              "pointValue": 2,
            },
          ]
    }
    discardedCards={[Function]}
    oneScore={0}
    teamNames={
        Array [
            "team1",
            "team2",
          ]
    }
    teamOneScore={[Function]}
    teamTimer="stopped"
    teamTwoScore={[Function]}
    twoScore={1}
    updateActiveCards={[Function]}
    updateTeamTimer={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "countDown": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "countDown": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`play matches the snapshot if activeCards.length === 0 && currRound <= 3 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Play
    activeCards={Array []}
    addCard={[Function]}
    currRound={1}
    currTeam="team2"
    currentRound={[Function]}
    currentTeam={[Function]}
    discardPile={
        Array [
            Object {
              "category": "ET CETERA",
              "description": "A virus that causes hemorrhagic fever, typically acquired by contact with an infected monkey, fruit bat, or person. Its most severe symptoms can include bleeding from the eyes, nose, ears, mouth, and rectum. There is no specific treatment for the disease.",
              "name": "Ebola",
              "pointValue": 2,
            },
            Object {
              "category": "CELEBRITY",
              "description": "A Japanese competitive eater who shocked the world in 2001 by eating 50 hot dogs and buns (HDB) in 12 minutes at Nathan's Hot Dog Eating Contest, doubling the previous record. He once lost a hot dog eating contest (no buns) to a 1089 lb. Kodiak bear (31-50).",
              "name": "Kobayashi",
              "pointValue": 2,
            },
          ]
    }
    discardedCards={[Function]}
    oneScore={0}
    teamNames={
        Array [
            "team1",
            "team2",
          ]
    }
    teamOneScore={[Function]}
    teamTimer="stopped"
    teamTwoScore={[Function]}
    twoScore={1}
    updateActiveCards={[Function]}
    updateTeamTimer={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "countDown": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "countDown": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`play matches the snapshot if currRound === 4 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Play
    activeCards={
        Array [
            Object {
              "category": "CELEBRITY",
              "description": "An Internet meme that shows a Shiba Inu surrounded by colorful Comic Sans text that describes its inner monologue, such as \\"Wow,\\" \\"Concern,\\" and \\"so scare.\\" There is much confuse over the name's pronunciation, yet it was recently used to brand a Bitcoin competitor.",
              "name": "Doge",
              "pointValue": 3,
            },
          ]
    }
    addCard={[Function]}
    currRound={4}
    currTeam="team1"
    currentRound={[Function]}
    currentTeam={[Function]}
    discardPile={
        Array [
            Object {
              "category": "ET CETERA",
              "description": "A virus that causes hemorrhagic fever, typically acquired by contact with an infected monkey, fruit bat, or person. Its most severe symptoms can include bleeding from the eyes, nose, ears, mouth, and rectum. There is no specific treatment for the disease.",
              "name": "Ebola",
              "pointValue": 2,
            },
            Object {
              "category": "CELEBRITY",
              "description": "A Japanese competitive eater who shocked the world in 2001 by eating 50 hot dogs and buns (HDB) in 12 minutes at Nathan's Hot Dog Eating Contest, doubling the previous record. He once lost a hot dog eating contest (no buns) to a 1089 lb. Kodiak bear (31-50).",
              "name": "Kobayashi",
              "pointValue": 2,
            },
          ]
    }
    discardedCards={[Function]}
    oneScore={0}
    teamNames={
        Array [
            "team1",
            "team2",
          ]
    }
    teamOneScore={[Function]}
    teamTimer="stopped"
    teamTwoScore={[Function]}
    twoScore={1}
    updateActiveCards={[Function]}
    updateTeamTimer={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {},
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`play matches the snapshot if currTeam !== teamNames[0] 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Play
    activeCards={
        Array [
            Object {
              "category": "CELEBRITY",
              "description": "An Internet meme that shows a Shiba Inu surrounded by colorful Comic Sans text that describes its inner monologue, such as \\"Wow,\\" \\"Concern,\\" and \\"so scare.\\" There is much confuse over the name's pronunciation, yet it was recently used to brand a Bitcoin competitor.",
              "name": "Doge",
              "pointValue": 3,
            },
          ]
    }
    addCard={[Function]}
    currRound={1}
    currTeam="team2"
    currentRound={[Function]}
    currentTeam={[Function]}
    discardPile={
        Array [
            Object {
              "category": "ET CETERA",
              "description": "A virus that causes hemorrhagic fever, typically acquired by contact with an infected monkey, fruit bat, or person. Its most severe symptoms can include bleeding from the eyes, nose, ears, mouth, and rectum. There is no specific treatment for the disease.",
              "name": "Ebola",
              "pointValue": 2,
            },
            Object {
              "category": "CELEBRITY",
              "description": "A Japanese competitive eater who shocked the world in 2001 by eating 50 hot dogs and buns (HDB) in 12 minutes at Nathan's Hot Dog Eating Contest, doubling the previous record. He once lost a hot dog eating contest (no buns) to a 1089 lb. Kodiak bear (31-50).",
              "name": "Kobayashi",
              "pointValue": 2,
            },
          ]
    }
    discardedCards={[Function]}
    oneScore={0}
    teamNames={
        Array [
            "team1",
            "team2",
          ]
    }
    teamOneScore={[Function]}
    teamTimer="stopped"
    teamTwoScore={[Function]}
    twoScore={1}
    updateActiveCards={[Function]}
    updateTeamTimer={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "countDown": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "countDown": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`play matches the snapshot if teamTimer === pregame 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Play
    activeCards={
        Array [
            Object {
              "category": "CELEBRITY",
              "description": "An Internet meme that shows a Shiba Inu surrounded by colorful Comic Sans text that describes its inner monologue, such as \\"Wow,\\" \\"Concern,\\" and \\"so scare.\\" There is much confuse over the name's pronunciation, yet it was recently used to brand a Bitcoin competitor.",
              "name": "Doge",
              "pointValue": 3,
            },
          ]
    }
    addCard={[Function]}
    currRound={1}
    currTeam="team1"
    currentRound={[Function]}
    currentTeam={[Function]}
    discardPile={
        Array [
            Object {
              "category": "ET CETERA",
              "description": "A virus that causes hemorrhagic fever, typically acquired by contact with an infected monkey, fruit bat, or person. Its most severe symptoms can include bleeding from the eyes, nose, ears, mouth, and rectum. There is no specific treatment for the disease.",
              "name": "Ebola",
              "pointValue": 2,
            },
            Object {
              "category": "CELEBRITY",
              "description": "A Japanese competitive eater who shocked the world in 2001 by eating 50 hot dogs and buns (HDB) in 12 minutes at Nathan's Hot Dog Eating Contest, doubling the previous record. He once lost a hot dog eating contest (no buns) to a 1089 lb. Kodiak bear (31-50).",
              "name": "Kobayashi",
              "pointValue": 2,
            },
          ]
    }
    discardedCards={[Function]}
    oneScore={0}
    teamNames={
        Array [
            "team1",
            "team2",
          ]
    }
    teamOneScore={[Function]}
    teamTimer="pregame"
    teamTwoScore={[Function]}
    twoScore={1}
    updateActiveCards={[Function]}
    updateTeamTimer={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "countDown": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "countDown": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`play matches the snapshot if teamTimer === stopped && currRound <= 3 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Play
    activeCards={
        Array [
            Object {
              "category": "CELEBRITY",
              "description": "An Internet meme that shows a Shiba Inu surrounded by colorful Comic Sans text that describes its inner monologue, such as \\"Wow,\\" \\"Concern,\\" and \\"so scare.\\" There is much confuse over the name's pronunciation, yet it was recently used to brand a Bitcoin competitor.",
              "name": "Doge",
              "pointValue": 3,
            },
          ]
    }
    addCard={[Function]}
    currRound={1}
    currTeam="team1"
    currentRound={[Function]}
    currentTeam={[Function]}
    discardPile={
        Array [
            Object {
              "category": "ET CETERA",
              "description": "A virus that causes hemorrhagic fever, typically acquired by contact with an infected monkey, fruit bat, or person. Its most severe symptoms can include bleeding from the eyes, nose, ears, mouth, and rectum. There is no specific treatment for the disease.",
              "name": "Ebola",
              "pointValue": 2,
            },
            Object {
              "category": "CELEBRITY",
              "description": "A Japanese competitive eater who shocked the world in 2001 by eating 50 hot dogs and buns (HDB) in 12 minutes at Nathan's Hot Dog Eating Contest, doubling the previous record. He once lost a hot dog eating contest (no buns) to a 1089 lb. Kodiak bear (31-50).",
              "name": "Kobayashi",
              "pointValue": 2,
            },
          ]
    }
    discardedCards={[Function]}
    oneScore={0}
    teamNames={
        Array [
            "team1",
            "team2",
          ]
    }
    teamOneScore={[Function]}
    teamTimer="stopped"
    teamTwoScore={[Function]}
    twoScore={1}
    updateActiveCards={[Function]}
    updateTeamTimer={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "countDown": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "countDown": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
